config/config.go

```diff
@@ -74,6 +85,10 @@ func New() (*Config, error) {
 	c.Core.Database.ExecutionRelativePath = filepath.Join("database", "executions")
 	c.Docker.Core.Path = "/mesg"
 	c.Docker.Socket = "/var/run/docker.sock"
+	c.Service.Foo = ServiceConfig{"https://github.com/mesg-foundation/service-ethereum-erc20", map[string]string{}}
+	c.Services = []ServiceConfig{
```

Comment:
in order to iterate on it and see the full list of services. This can be moved to the `core` but i would like to have it in the config like that we can also later on create this config api

Reply:
Done, I've created a function to access these services like that we keep it dynamic for the env injection